course7/homeworks/waleed/ls/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
If you're not using any feature that was introduced in CMake 3.9, then it's better yo use lower version (3.0 for instance), so that people who happen to have Cmake 3.8 also can build the project.